src/Tribe/Rewrite.php

```diff
@@ -487,11 +492,38 @@ public function remove_percent_placeholders( array $rules ) {
 
 			foreach ( $rules as $key => $value ) {
 				$key = str_replace( self::PERCENT_PLACEHOLDER, '%', $key );
-				$new_rules[$key] = $value;
+				$new_rules[ $key ] = $value;
 			}
 
 			return $new_rules;
 		}
+
+		/**
+		 * A function to Add an \ before the required caracters in Regex
+		 * Avoid using it elsewhere
+		 *
+		 * @param  string $value
+		 * @return string
+		 */
+		public function _build_replace_esc_regex( $value ) {
```

Comment:
Why the `_` prefix on the method name (ditto `_esc_regex_elements()`) ... might an `@internal` doc tag be better?